### ISSUE TITLE
Add labels to form elements in Advanced Search

### ISSUE THREAD

**Issue by axamei** [pulibrary/orangelight#1070]
Violation: Form elements must have labels.

Add `<label>` or aria-label or aria-labelledby

**Comment by tpendragon** [comment]
Closed by #1232
